.planning/codebase/STRUCTURE.md

```
@@ -1,268 +1,60 @@
 # Codebase Structure
 
-**Analysis Date:** 2026-06-04
+**Analysis Date:** 2026-07-18
 
 ## Directory Layout
-
 ```
```

markdownlint-cli2 (0.23.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)